ui/page.go

```diff
@@ -270,12 +276,23 @@ func (win *Window) addPages(decredIcons map[string]image.Image) {
 	win.pages[PageUTXO] = win.UTXOPage(common)
 	win.pages[PageAccountDetails] = win.AcctDetailsPage(common)
 	win.pages[PagePrivacy] = win.PrivacyPage(common)
+	win.pages[ValidateAddress] = win.ValidateAddressPage(common)
 }
 
 func (page pageCommon) ChangePage(pg string) {
 	page.changePage(pg)
 }
 
+func (page pageCommon) PushNavigationPage(from, to string) {
+	page.returnPage = &from
+	page.pushNavigationPage(from)
+	page.changePage(to)
+}
+
+func (page pageCommon) PopNavigationPage() {
```

Comment:
This method name is a bit misleading. Pop and push are commonly used for removing or adding data to an array, this method has no association to an array. You're simply setting values of the pages to a field in the common struct.